### PR TITLE
Update README.md

### LINKED ISSUE #1: Make the header of the readme match the Awesome standard
Additional details in the [Awesome list guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md)

image

### PR DESCRIPTION
**Pull request by yordanhristovUX**
Logo and badge added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-# awesome-hicetnunc
-Awesome list of things in the broader hic et nunc ecosystem
+# awesome-hicetnunc [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+<img align="right" width="100" height="100" src="https://raw.githubusercontent.com/yordanhristovUX/awesome-hicetnunc/bb2bac00cb8a36d2f9e541cd7ff56a5415e82eae/logoawehen.svg">
```
Comment (0x00000010) [inline README.md]: now that #11 is merged please point this url to the relative local location of the logo

thank you!

**Review by 0x00000010 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (0x00000010) [reply README.md]: please dm me on twitter / discord :)

Comment (yordanhristovUX) [reply README.md]: Hi, was not on my PC, I think I put relative location, but not sure if it is right. Twitter don't let me message you and I don't have you discord. I am @brutalisti at Twitter, probably if you follow me, I will be able to message.

**Review by 0x00000010 — APPROVED** [review]

**Comment by 0x00000010** [comment]
Resolves #1

**Pull request merged**